documentation/IDTA-01001/modules/ROOT/pages/spec-metamodel/designators.adoc

```diff
@@ -20,4 +20,4 @@ To understand the specifications, it is crucial to understand the common attribu
 
 The concept of referencing and how a reference is represented in the UML diagrams and the tables is explained in Clause 5.3.9 and Annex E.
 
-Constraints that are no invariants of classes are specified in Clause 5.3.11.3.
+Constraints that are no invariants of classes are specified in Clause xref:constraints.adoc[Constraints: Global Invariants].
```

Comment:
The [file coordinate](https://docs.antora.org/antora/latest/page/resource-id-coordinates/#id-resource) of a target page is calculated from the pages family directory [[Source](https://docs.antora.org/antora/latest/page/xref/#structure)]
```suggestion
Constraints that are no invariants of classes are specified in Clause xref:spec-metamodel/constraints.adoc[Constraints: Global Invariants].
```